pulp_ansible/app/tasks/collections.py

```diff
@@ -503,6 +512,27 @@ async def run(self):
         """
         tasks = []
         loop = asyncio.get_event_loop()
+        root, api_version = await self._get_collection_api_root(self.remote.url)
+        if api_version == 3:
+            downloader = self.remote.get_downloader(
+                url=root, silence_errors_for_response_status_codes={404}
+            )
+            try:
+                metadata = parse_metadata(await downloader.run())
+            except FileNotFoundError:
+                pass
+
+            collection_info = self.collection_info or []
+            sources = {r.source for r in collection_info if r.source}
+            sources.add(self.remote.url)
+
+            published = metadata.get("published")
+            if published == self.remote.last_synced_metadata_time and len(sources) < 2:
+                log.info(_("no-op"))
```

Comment:
Can this log message be expanded some. It would be good if it also logged if it's being no-opped due to too many sources, versus the published time not changing.